docs/userguide/application.rst

```diff
@@ -475,16 +468,16 @@ chain breaks:
 
     .. code-block:: python
 
-        from celery.task import task
+        from celery import app
 
-        @task(queue='hipri')
+        @app.task(queue='hipri')
         def hello(to):
             return 'hello {0}'.format(to)
 
 Abstract Tasks
 ==============
 
-All tasks created using the :meth:`~@task` decorator
+All tasks created using the :meth:`~@app.task` decorator
```

Comment:
Does this render properly?

Reply:
It does render properly, just tested it.

Reply:
Yeah, I stole that from elsewhere in the docs. :grin: